subprojects/gst-plugins-good/sys/v4l2/gstv4l2allocator.c

```diff
@@ -1023,6 +1029,97 @@ gst_v4l2_allocator_alloc_dmabufin (GstV4l2Allocator * allocator)
   return group;
 }
 
+GstV4l2MemoryGroup *
+gst_v4l2_allocator_alloc_dmabufin_capture (GstV4l2Allocator * allocator,
+    GstAllocator * dmabuf_allocator, GstBuffer* downstream_buffer)
+{
+#if 1
```

Comment:
Don't leave personal debugging code please.

Reply:
Removed it.